tools/make/kube.mk

```diff
@@ -102,12 +102,15 @@ install-ratelimit:
 	@$(LOG_TARGET)
 	kubectl apply -f examples/redis/redis.yaml
 	kubectl rollout restart deployment envoy-gateway -n envoy-gateway-system
+	kubectl rollout status --watch --timeout=5m -n envoy-gateway-system deployment/envoy-gateway
```

Comment:
why is this needed ?

Reply:
the next line checks for `condition=Available`

Reply:
because the `rollout restart` does not affect the condition of deployments.

even through its replicaset is not ready yet, the condition of deployments is still `Available`. so the `rollout status` here watches till its replicaset gets ready.

Reply:
thanks for the clarification, seems weird that it doesnt reset the status